openassessment/xblock/job_sample_grader/code_grader.py

```diff
@@ -239,6 +283,14 @@ def run_code(self, run_type, executor_id, source_code, problem_name):
                     'correct': run_output['correct'],
                 }
 
+                input_file_name = case_file['input_file']['name']
+                output_file_name = case_file['expected_output_file']['name']
+
+                if question_mapping and os.path.isfile(input_file_name):
+                    os.remove(input_file_name)
+                if question_mapping and os.path.isfile(output_file_name):
+                    os.remove(output_file_name)
+
```

Comment:
Won't this delete the grader data in the case when a block mapping does not exist?